hedera-node/src/main/java/com/hedera/services/ledger/HederaLedger.java

```diff
@@ -150,6 +155,9 @@ public HederaLedger(
 		this.accountsLedger = accountsLedger;
 		this.dynamicProperties = dynamicProperties;
 		this.sideEffectsTracker = sideEffectsTracker;
+		this.tokensLedger = tokensLedger;
+
+		this.transferLogic = new TransferLogic(accountsLedger, nftsLedger, tokenRelsLedger, tokenStore, sideEffectsTracker, tokenViewsManager, dynamicProperties, validator);
```

Comment:
Instead of creating new instance of the `TransferLogic` can't we just inject it using `dagger`?